dice-game-ui-angular/src/app/statistics/statistics.component.ts

```diff
@@ -17,19 +17,24 @@ export interface StatisticsDTO {
   styleUrls: ['./statistics.component.scss'],
 })
 export class StatisticsComponent implements OnInit {
-  statistics: StatisticsDTO[];
+  private user: SocialUser;
+  statistics: StatisticsDTO[] = [];
   displayedColumns: string[];
 
-  constructor(private statisticsService: StatisticsService) {
+  constructor(private authService: SocialAuthService, private statisticsService: StatisticsService) {
     this.displayedColumns = ['wasCompleted', 'gameType', 'fieldSize', 'turnsCount', 'score', 'isWinner'];
   }
 
   ngOnInit(): void {
-    this.getAllStatistics();
+    this.getUserStatistics();
```

Comment:
Method with word `get` in the name should return something. If this method just initializes some variables, it should better be named `load...`.

Reply:
Changed